tools/internal_ci/linux/grpc_xds_url_map.sh

```diff
@@ -13,13 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -ex -o igncr || set -ex
+set -eo pipefail
 
 # Constants
 readonly GITHUB_REPOSITORY_NAME="grpc"
-# GKE Cluster
-readonly GKE_CLUSTER_NAME="interop-test-psm-sec-v2-us-central1-a"
```

Comment:
TIL This still used to use PSM Sec cluster. That's why we need these backports, and `activate_gke_cluster` feature. Thanks, Terry.
cc @lidizheng

Reply:
Thanks for updating the cluster name!